pkg/config/jobs.go

```diff
@@ -266,6 +266,19 @@ type Postsubmit struct {
 	JenkinsSpec *JenkinsSpec `json:"jenkins_spec,omitempty"`
 }
 
+// RetriggerFailed defines the configuration for retrying failed prowjobs.
+type RetriggerFailed struct {
+	// UntilSuccess stops retries once a successful run is achieved.
+	// The Attempts field still limits the total number of retries.
+	RunAll bool `json:"run_all,omitempty"`
+	// Attempts specifies the maximum number of retry attempts allowed.
+	Attempts int `json:"attempts,omitempty"`
+	// Interval defines the wait duration between consecutive retry attempts.
+	Interval string `json:"interval,omitempty"`
```

Comment:
Interval is good but we could add a `backoff` configuration as well. I am not sure that the interval time will always be accurate between the retries. Shall we also consider how horologium reconciles the prowjobs?

Reply:
I do not think backoff is needed in this case. Let's take somebody who has test that runs 1h10m but defined 1h interval. In the current implementation, this periodic will be just scheduled right after previous one stops. 
The only case that this not happen is when we have repetition of "big interval" and the next queue is about to start, but the old one is not finished. So:

1st run 2nd run ... should be 3rd but is taking so long so instead 1st run occurs.

I think that does not require changes in implementation but a proper comment explanation to document the chosen approach. If that would be the only nit, I will update comment as a followup.

Reply:
specifically speaking about second part of https://github.com/kubernetes-sigs/prow/pull/358#pullrequestreview-2559293998

Reply:
documented behavior not happened and my proposal is to do it as followup (if that ends to be the only ask)